utils.py

```diff
@@ -2125,47 +2135,37 @@ def num_denom(A, symbolic_expr=False):
         denom = [numpy.poly1d(denom) *
                  numpy.poly1d(A.matrix[0, j].denominator.coeffs)
                  for j in range(A.matrix.shape[1])]
-        num = [numpy.poly1d(num) *
-               numpy.poly1d(A.matrix[0, j].numerator.coeffs)
+        num = [numpy.poly1d(num)
+               * numpy.poly1d(A.matrix[0, j].numerator.coeffs)
                for j in range(A.matrix.shape[1])]
-        if symbolic_expr is True:
-            for n in range(len(denom)):
-                sym_den = (sym_den + denom[- n - 1] * s**n).simplify()
-            for n in range(len(num)):
-                sym_num = (sym_num + num[- n - 1] * s**n).simplify()
-            return sym_num, sym_den
-        else:
-            return num, denom
-
     elif type(A) == tf:
-        denom = []
-        num = []
 
         denom = [list(A.denominator.coeffs)[n] for n in range(
             len(list(A.denominator.coeffs)))]
         num = [list(A.numerator.coeffs)[n] for n in range(
             len(list(A.numerator.coeffs)))]
-        if symbolic_expr is True:
-            for n in range(len(denom)):
-                sym_den = (sym_den + denom[- n - 1] * s**n).simplify()
-            for n in range(len(num)):
-                sym_num = (sym_num + num[- n - 1] * s**n).simplify()
-            return sym_num, sym_den
-        else:
-            return num, denom
-"""
+
+    if symbolic_expr is True:
+        for n in range(len(denom)):
+            sym_den = (sym_den + denom[- n - 1] * s**n).simplify()
+        for n in range(len(num)):
+            sym_num = (sym_num + num[- n - 1] * s**n).simplify()
+        return sym_num, sym_den
     else:
-        sym_num, sym_den = A.as_numer_denom()
-        if not symbolic_expr:
-            num_poly   = sympy.Poly(sym_num)
-            numer      = [float(k) for k in num_poly.all_coeffs()]
-            den_poly   = sympy.Poly(sym_den)
-            denom      = [float(k) for k in den_poly.all_coeffs()]
-            return numer, denom
-        else:
-            return sym_num, sym_den
-"""
+        return num, denom
 
+    """
```

Comment:
I dislike using multiline strings as comments. Also, In general it is better to delete things that are commented out from the repository. If you feel it is still useful, either activate the code or log an issue, but commented code is actually dead.

Reply:
Alright, I wasn't sure if you'd want it left there or not. I'll remove commented code in future reviews.